Dockerfiles/agent/entrypoint/10-dogstatsd-socket.sh

```diff
@@ -2,13 +2,13 @@
 
 # Delete the dogstatsd unix socket if present
 # FIXME: move that logic to dsd itself
-if [[ -e "${DD_DOGSTATSD_SOCKET}" ]]; then
-    if [[ -S "${DD_DOGSTATSD_SOCKET}" ]]; then
-        echo "Deleting existing socket at ${DD_DOGSTATSD_SOCKET}"
-        rm -v "${DD_DOGSTATSD_SOCKET}" || exit $?
+if [[ -e "${STS_DOGSTATSD_SOCKET}" ]]; then
+    if [[ -S "${STS_DOGSTATSD_SOCKET}" ]]; then
+        echo "Deleting existing socket at ${STS_DOGSTATSD_SOCKET}"
+        rm -v "${STS_DOGSTATSD_SOCKET}" || exit $?
     else
```

Comment:
Can we perhaps call this something different than `DOGSTATSD_SOCKET`.

Reply:
will discuss with mark, we currently dont fork this project i think, maybe make it a separarte story

Reply:
discussed with mark, this is for later